C/tests/c4QueryTest.cc

```diff
@@ -985,6 +985,48 @@ N_WAY_TEST_CASE_METHOD(NestedQueryTest, "C4Query Nested UNNEST", "[Query][C]") {
     }
 }
 
+N_WAY_TEST_CASE_METHOD(NestedQueryTest, "C4Query Nested UNNEST - Missing Array", "[Query][C]") {
+    deleteDatabase();
+    db = c4db_openNamed(kDatabaseName, &dbConfig(), ERROR_INFO());
+    std::stringstream documents{R"(
+[
+{"name": "Jonh Doe", "contacts": ["Contact1", "Contact2"]},
+{"name": "Scott Tiger", "contacts": []},
+{"name": "Foo Bar"},
+{"name": "Harry Potter", "contacts": "my contacts" }
+]
+)"};
```

Comment:
The indentation appears to be incorrect here.

Reply:
will fix